app/controllers/sitemaps_controller.rb

```diff
@@ -1,7 +1,7 @@
 class SitemapsController < ApplicationController
   def index
     @host = "#{request.protocol}#{request.host}"
-    @steps = StepFactory::STEP_NAMES
+    @steps = Base.descendants.map(&:to_s)
```

Comment:
This would include all child classes, irrespective of whether they are abstract? Whats the advantage of removing the list of what should be there.

Separately - do we actually want to include all the steps in the sitemap?